pom.xml

```diff
@@ -145,6 +145,11 @@
             <artifactId>validation-api</artifactId>
         </dependency>
         <dependency>
+            <groupId>com.google.code.findbugs</groupId>
+            <artifactId>annotations</artifactId>
+            <version>3.0.0</version>
```

Comment:
> maybe let's do what findbugs is asking us to do?

@yegor256 Tried that, still warns. It's a genuine false positive; see https://github.com/jcabi/jcabi-github/pull/1120#discussion_r32302694